### PR TITLE
docs/variants/novacustom_ns5x_7x/: refactor documentation

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/variants/novacustom_ns5x_7x/building-manual.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
-# Dasharo for NovaCustom NS5X - Building manual
+# Dasharo for NovaCustom NS5x/7x - building manual
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_ns5x_7x/building-manual.md:1]: In the title, we should limit ourselves only to section name, i.e. `Building manual`

File: docs/variants/novacustom_ns5x_7x/overview.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Overview
+
+* [Releases](releases.md) - list of all releases
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_ns5x_7x/overview.md:3]: In section `Overview` we should use descriptions as in other refactored documentation, i. e. ASUS KGPE-D16.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_ns5x_7x/building-manual.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/variants/novacustom_ns5x_7x/building-manual.md]: I updated also building-manual.md for Asus and Talos

File: docs/variants/novacustom_ns5x_7x/releases.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ See how to verify signatures on [this video](https://asciinema.org/a/433461)
 
 ### Added
 
-- Support for NovaCustom NS5x
+- Support for NovaCustom NS5x/7x
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_ns5x_7x/releases.md:96]: This is not the truth. Support for NS7x has been added in v1.1.0 of the Dasharo firmware

Comment (Pokisiekk) [reply docs/variants/novacustom_ns5x_7x/releases.md]: Updated

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**